generator/src/main/java/org/apache/kafka/message/MessageDataGenerator.java

```diff
@@ -1901,7 +1901,9 @@ private void generateFieldToString(String prefix, FieldSpec field) {
             buffer.printf("+ \"%s%s=\" + %s%n",
                     prefix, field.camelCaseName(), field.camelCaseName());
         } else if (field.type().isStruct() ||
-            field.type() instanceof FieldType.UUIDFieldType) {
+                field.type() instanceof FieldType.UUIDFieldType) {
+            buffer.printf("+ \"%s%s=\" + %s.toString()%n",
+                prefix, field.camelCaseName(), field.camelCaseName());
         } else if (field.type().isStruct()) {
```

Comment:
Should this line be removed if it is covered by the line above?

Reply:
good point.  I will consolidate these.